### ISSUE TITLE
The art for the tokens created by [Godsire] is incorrect.

### ISSUE THREAD

**Issue by BackAlleyG** [PennyDreadfulMTG/modo-bugs#291]
**Screenshot**
![image](https://user-images.githubusercontent.com/31076829/35660639-342e6ad0-06db-11e8-9dc6-eff2fad6bf58.png)
![image](https://user-images.githubusercontent.com/31076829/35660677-75c91d6e-06db-11e8-8de5-2744ff453c72.png)

**Please provide a list of affected cards**
Affects: [Godsire]

**Card Images**

**Comment by BackAlleyG** [comment]
Something messed up the token/token art system as a whole pretty bad. I'm not sure what caused it but I remember for sure the Godsire tokens being correct not that long ago from playing momir basic a decent amount, maybe as far back as HOD if not more recently (which is likely). For now, I'm sticking with my theory that FTV: Transform potentially cause the issue.

**Comment by Rowsol** [comment]
Yeah, token art is all over the place.  Flip Garruk's deathtouch wolf tokens are [Sound the Call] tokens.
